modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,307 @@
++
+[NOTE]
+====
+* If add-on is created using the `TektonConfig` custom resource, and you change the parameter values later in the
+`Addon` custom resource, then the changes will be overwritten by the values in the `TektonConfig` custom resource.
+
+* The value of the `pipelinesTemplates` parameter can be `true` only when the value of the `clusterTasks` parameter is `true`.
+====
+
+* The `enableMetrics` parameter is added to the `TektonConfig` custom resource. You can use it to disable the service monitor, which is part of Tekton Pipelines for {product-title}.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  profile: all
+  targetNamespace: openshift-pipelines
+  pipeline:
+    params:
+    - name: enableMetrics
+      value: "true"
+...
+----
+
+* Eventlistener OpenCensus metrics, which captures metrics at process level, has been added.
+
+* Triggers now has label selector; you can configure triggers for an event listener using labels.
+
+* The `ClusterInterceptor` custom resource definition for registering interceptors is added, which allows you to register new `Interceptor` types that you can plug in. In addition, the following relevant changes are made:
+
+** In the trigger specifications, you can configure interceptors using a new API that includes a `ref` field to refer to a cluster interceptor. In addition, you can use the `params` field to add parameters that will be passed on to the interceptors for processing.
+
+** The bundled interceptors CEL, GitHub, GitLab, and BitBucket, have been migrated. They are implemented using the new `ClusterInterceptor` custom resource definition.
+
+** Core interceptors are migrated to the new format, and any new triggers created using the old syntax will automatically switch to the new `ref` or `params` based syntax.
+
+* To disable prefixing the name of the task or step while displaying logs, you can use the `--prefix` option for `log` commands.
+
+* To display the version of a specific component, use the new `--component` flag in the `tkn version` command.
+
+* The `tkn hub check-upgrade` command is added, and other commands are revised to be based on the pipeline version. In addition, catalog names are displayed in the `search` command output.
+
+* Support for optional workspaces are added to the `start` command.
+
+* The plugins will be looked up in the current path, if they are not present in the `plugins` directory.
+
+* The `tkn [task | clustertask | pipeline] start` command will start interactively and ask for the `params` value, if default is not specified or passed through flag.
+
+* The `version` field is added in the `tkn task describe` command.
+
+* The option to automatically select `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener`, has been added in the `describe` command, if only one is present.
```

Comment:
select `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener` --> specify the kind o fthe object

For example: select the resource: `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener`

Reply:
if only one is present: is it the option or the resource? Unclear. Please rewrite.